spec/features/users_can_create_involuntary_conversion_projects_spec.rb

```diff
@@ -13,7 +13,7 @@
     let(:ukprn) { 10061021 }
     let(:two_weeks_ago) { Date.today - 2.weeks }
 
-    before { mock_successful_api_responses(urn: urn, ukprn: ukprn) }
+    before { mock_successful_api_responses(urn: any_args, ukprn: any_args) }
```

Comment:
Just had a poke at this and it looks like URN and UKPRN are no longer cast to integers in the model, but we still need to set up the mock for the factory as well.

I think one option is to change from using attr_accessor to using the `attribute` method of `ActiveModel::Attributes` so that we can cast values, and have:

```ruby
include ActiveModel::Attributes

attribute :urn, :integer
```

If not, I think this will have to do!

Reply:
I tried this when doing the work originally (I saw you'd used it in a closed PR) but I am a bit baffled by it, I can't figure out why I get this error: 

```Failure/Error: attribute :urn, :integer

NameError:
  undefined local variable or method `attribute_aliases' for Conversion::CreateProjectForm:Class

          name = attribute_aliases[name] || name
                 ^^^^^^^^^^^^^^^^^
  Did you mean?  attribute
```

I don't know if it's so important to figure this out now or not?

Reply:
It should just need the mixins:

  include ActiveModel::Model
  include ActiveModel::Attributes
  
Other than that, I'm not sure!

Reply:
FTR this was me being a new year, first day back muppet and typing `include ActiveRecord::Attributes` 🙄